slides/annotations.qmd

```diff
@@ -264,3 +264,28 @@ Effect encoding might result in a somewhat circular argument: the column is more
 
 Partial pooling somewhat lowers the risk of overfitting since it tends to correct for agents with small sample sizes. It can’t correct for improper data usage or data leakage though. 
 
+# Extras - Case Study on Transportation
+
+## A recipe - handle correlations `r hexes(c("recipes"))`
+
+In this code chunk: 
+
+
+```{r}
+#| label: pca-rec
+#| eval: false
+chi_pca_rec <- 
+  chi_rec %>% 
+  step_normalize(all_of(!!stations)) %>% 
+  step_pca(all_of(!!stations), num_comp = tune())
+```
+
+What's the story with `!!stations`? 
```

Comment:
I would put this in the line above the code chunk so that people know what to look for when reading the code chunk